src/databricks/labs/ucx/account.py

```diff
@@ -69,6 +71,77 @@ def sync_workspace_info(self):
             for installation in Installation.existing(ws, "ucx"):
                 installation.save(workspaces, filename=self.SYNC_FILE_NAME)
 
+    def create_account_level_groups(self):
+        acc_groups = self._get_account_groups()
+        all_valid_workspace_groups = self._get_valid_workspaces_groups()
+
+        for group_name, valid_group in all_valid_workspace_groups.items():
+            if group_name in acc_groups:
+                logger.info(f"Group {group_name} already exist in the account, ignoring")
+            else:
+                acc_group = self._ac.groups.create(display_name=group_name)
+
+                if len(acc_group.members) > 1:
+                    self._add_members_to_acc_group(acc_group.id, group_name, valid_group)
+
+                logger.info(f"Group {group_name} created in the account")
+
+    def _add_members_to_acc_group(self, acc_group_id: str, group_name: str, valid_group: Group):
+        for chunk in self._chunks(valid_group.members, 20):
+            logger.debug(f"Adding 20 members to acc group {group_name}")
+            self._ac.groups.patch(
+                acc_group_id,
+                operations=[iam.Patch(op=iam.PatchOp.ADD, path="members", value=[x.as_dict() for x in chunk])],
+                schemas=[iam.PatchSchema.URN_IETF_PARAMS_SCIM_API_MESSAGES_2_0_PATCH_OP],
+            )
+
+    def _chunks(self, lst, n):
+        """Yield successive n-sized chunks from lst."""
+        for i in range(0, len(lst), n):
+            yield lst[i : i + n]
+
+    def _get_valid_workspaces_groups(self) -> dict[str, Group]:
+        all_workspaces_groups: dict[str, Group] = {}
+
+        for client in self.workspace_clients():
+            ws_group_ids = client.groups.list(attributes="id")
+            for group_id in ws_group_ids:
+                if not group_id.id:
+                    continue
+                full_workspace_group = client.groups.get(group_id.id)
+                group_name = full_workspace_group.display_name
+
+                if group_name in all_workspaces_groups:
+                    if self._has_not_same_members(all_workspaces_groups[group_name], full_workspace_group):
+                        logger.warning(
+                            f"Group {group_name} does not have same amount of members "
+                            f"in workspace {client.config.host}, it will be created with account "
+                            f"name {client.config.host}_{group_name}"
+                        )
+                        all_workspaces_groups[f"{client.config.host}_{group_name}"] = full_workspace_group
```

Comment:
Can you use workspace name instead of the hostname? See account.py

I also think that you should prompts.confirm() that with the user

Reply:
We should make a separate to PR to handle multiple scenarios, let's create a default way to handle this first. Workspace name is indeed much better